interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is javaScript lenguage concatenated with HTLM by the React app to effectively write code.
 
-Researched answer:
+Researched answer: JSX is a syntax similart to HTML in react to for easier use by developers. HTML tells the browser how to display its elements.
```

Comment:
What is one syntax difference?

Reply:
Will be looking for a much-researched answer and provide a better answer with syntax examples on this comment threat.